src/config/dynamic-theme-fixes.config

```diff
@@ -5,6 +5,15 @@ svg
 
 ================================
 
+elearning.utdallas.edu/webapps/portal/execute/tabs/tabAction?tab_tab_group_id=_1_1
+NO
```

Comment:
We have `INVERT` and `NO INVERT` commands, `NO` does nothing

Reply:
That's what I kept trying, but every time I would hit apply, the developer tool would move the NO up there for some reason. I'll move it back down like it's supposed to be.